src/components/search/index.js

```diff
@@ -86,18 +42,25 @@ export default function Search({ collapse }) {
   )
 
   return (
-    <div className="search-root" ref={ref}>
+    <div className="relative flex-grow flex items-center" ref={ref}>
       <InstantSearch
         searchClient={searchClient}
         indexName={indices[0].name}
         onSearchStateChange={({ query }) => setQuery(query)}
-        root={{ Root, props: { ref } }}
-        styles={{ position: "relative" }}
       >
         <Configure hitsPerPage={8} />
         <Input onFocus={() => setFocus(true)} {...{ collapse, focus }} />
         {searchQuery?.length > 0 && focus && (
-          <div className="hits-wrapper">
+          <div
+            className="grid absolute left-0 overflow-hidden border border-solid bg-white"
+            style={{
```

Comment:
Styles can't be mapped to default tailwindcss classes are written in `style` prop, we can easily refactor them when `tailwind.config.js` is standardize later.

Reply:
@chenxsan I think most of these are fine to be mapped to Tailwind approximates–they were never considered the intended design as much as they were copy pasted. Let's try using the closest approximates for zIndex, borderColor, borderRadius.